src/build/functions/edge.ts

```diff
@@ -85,13 +85,27 @@ const writeHandlerFile = async (ctx: PluginContext, { matchers, name }: NextDefi
     JSON.stringify(minimalNextConfig),
   )
 
+  const htmlRewriterWasm = await readFile(
+    join(
+      ctx.pluginDir,
+      'edge-runtime/vendor/deno.land/x/htmlrewriter@v1.0.0/pkg/htmlrewriter_bg.wasm',
+    ),
+  )
+
   // Writing the function entry file. It wraps the middleware code with the
   // compatibility layer mentioned above.
   await writeFile(
     join(handlerDirectory, `${handlerName}.js`),
     `
+    import { decode as _base64Decode } from './edge-runtime/vendor/deno.land/std@0.175.0/encoding/base64.ts';
+    import { init as htmlRewriterInit } from './edge-runtime/vendor/deno.land/x/htmlrewriter@v1.0.0/src/index.ts'
     import {handleMiddleware} from './edge-runtime/middleware.ts';
     import handler from './server/${name}.js';
+
+    await htmlRewriterInit({ module_or_path: _base64Decode(${JSON.stringify(
+      htmlRewriterWasm.toString('base64'),
+    )}).buffer });
```

Comment:
we could use a uint8array instead of a base64 string here, it would save on memory usage by 33%

I did this in a similar package I made specifically for CSP support - I could do the same for the htmlrewriter package if we want:
```js
await Deno.writeTextFile(
    "./pkg/embedded-wasm.ts",
    `export const wasmBinary = Uint8Array.from(${
      JSON.stringify(
        Array.from(
          await Deno.readFile(
            `./pkg/${wasmFile}`,
          ),
        ),
      )
    });`,
  );
 ```

https://github.com/netlify/csp_nonce_html_transformer/blob/main/scripts/build.ts#L13C3-L25